### DISCUSSION TITLE
Apphooks and placeholder template tags

### DISCUSSION THREAD

**Discussion by careercoder** [django-cms/django-cms#7112] [General]
Why was the choice made to not allow the content editor on subpages of an application (apphooks)?

I wish to understand the reason behind this. I feel like this is a big hold back.

**Comment by benzkji** [comment]
as far as I can say, only placeholders belonging to a page, are not working. If you create a PlaceholderField on your model, you can use the whole editing mechanism right there...? https://docs.django-cms.org/en/latest/how_to/placeholders.html

I've used page placeholders on app subpages, for translation of texts and stuff, but it always felt like a hack ;-)
